src/config/index.js

```diff
@@ -1,8 +1,7 @@
-import getConfig from "next/config";
 import { getLogger } from "../utils/logger";
 
 const { trace } = getLogger("config");
-const { publicRuntimeConfig } = getConfig();
+const config = { network: process.env.NET };
```

Comment:
Hmm actually quite weird to have a `config` variable in a config file right? Do we want to declare as `const NETWORK = process.env.NET` instead?

Reply:
done